tests/integration/workflows/fixtures/oai_arxiv_record_with_plots.xml

```diff
@@ -24,7 +24,7 @@
             </author>
           </authors>
           <title>Spectra generated by a confined softcore Coulomb potential</title>
-          <categories>math-ph math.MP</categories>
+          <categories>math-ph math.MP astro-ph astro-ph.IM</categories>
```

Comment:
That is true, but it's not taking into account:

* That it does not apply to this PR, specially to this record, as it's a 'smoke' record for a smoke test, that runs just one flow with one simple, valid input.

* A crafted record does not have to be valid or 'nice', actually, I'd say that crafting a record you cam make it uglier than you'd never get.

* The ugliness that is going to hit us later, by definition, will not be caught by the records we already have.

* All existing records, are a subset of all crafted records too, so anything you can do with an existing record, you can do with a crafted one, but not otherwise (see previous two points).

In essence, you can use crafted or non-crafted records, as long as you are choosing them wisely, though with crafted records you have a lot more power on how and what to test, including the ability to create records that will test several things at the same time, that test specific issues, reducing the amount of examples and passes you have to do to test all the cases, and the time it takes to find the real records that will trigger those cases.